tests/integration/annotation_import/test_label_import.py

```diff
@@ -38,6 +38,41 @@ def test_create_from_objects(client, configured_project, object_predictions,
         label_import.input_file_url, object_predictions)
 
 
+def test_data_row_validation_errors(client, configured_project,
```

Comment:
arguable, these do not need to be in integration tests, since they are only testing sdk-side logic